dist/jsoneditor.js

```diff
@@ -1715,7 +1715,15 @@ JSONEditor.AbstractEditor = Class.extend({
     return null;
   },
   getTitle: function() {
-    return this.schema.title || this.key;
+      //Title collapsable form sections
+      if (this.hasOwnProperty("hide_add_button")){
```

Comment:
unfortunately this ugly check is necessary because unless `hide_add_button` or a cousin was present we would not want the title info from parent